src/tool_roslyn/tool_roslyn.csproj

```diff
@@ -12,7 +12,7 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="$(CLI_Roslyn_Version)" />
     <PackageReference Include="Microsoft.CodeAnalysis.Build.Tasks" Version="$(CLI_Roslyn_Version)" />
     <PackageReference Include="Microsoft.Net.Compilers.netcore" Version="$(CLI_Roslyn_Version)" />
-    <PackageReference Include="Microsoft.DiaSymReader.Native" Version="1.4.1" />
+    <PackageReference Include="Microsoft.DiaSymReader.Native" Version="1.6.0-beta2-25304" />
```

Comment:
Right now RTM version is gonna be the final 15.3 VS build. If we need one earlier then we need to arrange it with C++ team, so that they produce, archive and service another build.